### PR TITLE
newsletter 2019-06-28

### PR DESCRIPTION
**Pull request by ryan-shea**

### CODE REVIEW

File: app/retail/emails.py [hunk 1]
```diff
@@ -932,8 +932,8 @@ def render_start_work_applicant_expired(interest, bounty):
 def render_new_bounty_roundup(to_email):
     from dashboard.models import Bounty
     from django.conf import settings
-    subject = "Beyond Blockchain Begins!"
-    new_kudos_pks = [4033, 4031, 4028]
+    subject = "A Hackaton: Continued"
```
Comment (owocki) [inline app/retail/emails.py]: Hackaton is misspelled

Comment (owocki) [reply app/retail/emails.py]: how about `Hacking Beyond Blockchain 💻🤓` instead?

File: app/retail/emails.py [hunk 2]
```diff
@@ -948,14 +948,15 @@ def render_new_bounty_roundup(to_email):
 Hey Gitcoiners,
 </p>
 <p>
-Our first hackathon of the summer season is here. Beyond Blockchain, a hackathon organized alongside Consensys Labs, focuses on the future. Over $15k of prizes will be available for this three week hackathon, and the hacking begins on Monday, June 24th, and goes until July 10th.
-For more info, read the announcement blog post <a href="https://gitcoin.co/blog/beyond-blockchain-hackathon/" target="_blank"> here </a> and join our Discord <a href="https://discord.gg/T98dQYC" target="_blank"> here! </a>
+Beyond Blockchain has officially begun! We kicked off the three week haul on Monday, and there's still plenty of time to vie for some of the prizes we have available. Our sponsors -- Consensys Labs, Raiden, Arweave, Portis, and The Graph -- have dozens of prizes ranging up to 22 ETH. Join our <a href="https://docs.google.com/document/d/1N8EcHOP7pzuvoPNvJ3VPBtv9KFJf9Ot1kjGU381Dh_s/edit?usp=sharing" target="_blank">Discord</a> and discuss the hackathon,
```
Comment (owocki) [inline app/retail/emails.py]: what is `https://docs.google.com/document/d/1N8EcHOP7pzuvoPNvJ3VPBtv9KFJf9Ot1kjGU381Dh_s/edit?usp=sharing` ? that doesnt seem like a link to the discord.  in fact, its a permissioned gdoc

File: app/retail/emails.py [hunk 3]
```diff
@@ -948,14 +948,15 @@ def render_new_bounty_roundup(to_email):
...
 </p>
 <p>
-Need to find a team for the hackathon? <a href="https://github.com/gitcoinco/beyondblockchainteams/issues">Post a profile in this GitHub repo</a> and mingle with others. We're here to help you find a crew.
+Our sponsors each offer a unique product and set of goals for their hackathon prompts. Read about their projects and their goals in <a href="https://gitcoin.co/blog/beyond-blockchain-the-players/?preview=true" target="_blank">this</a> blog post, and head on over to the GitHub page to read individual
+prompts. 
 </p>
 <p>
-Missed a livestream but still want to soak in the knowledge? We're way ahead of you. Check out <a href="https://youtube.com/GitcoinMedia" target="_blank">Gitcoin Media</a>, our official YouTube channel, for livestream content and more. We'll be updating it in the coming weeks so all of the livestreams are uploaded for content consumption goodness.
+Need to find a team for the hackathon to ease your workload? <a href="https://github.com/gitcoinco/beyondblockchainteams/issues">Post a profile in this GitHub repo</a> and mingle with others. We're here to help you find a crew.
```
Comment (owocki) [inline app/retail/emails.py]: Are you a Coder, and do you need to find a team for the hackathon?

Are you a Hirer, and do you need to find a coder to work on your project?

**Comment by owocki** [comment]
@coderberry do we want to update the codefund ad in this email for this week?  do we want to disable it until we find a paying customer?

**Comment by ryan-shea** [comment]
https://github.com/gitcoinco/web/pull/4705

**Pull request closed without merging**